packages/app/src/app/store/modules/workspace/actions.js

```diff
@@ -60,7 +100,14 @@ export function addTag({ api, path, state }) {
   };
   return api
     .post(`/sandboxes/${sandboxId}/tags`, body)
-    .then(data => path.success({ data }))
+    .then(data =>
+      path.success({
```

Comment:
I'm not sure if this is relevant. This is about tags, right?

Reply:
fixed

Reply:
fixed